content/system.md

```diff
@@ -1,10 +1,77 @@
 ---
 title: "System"
-date: 2022-07-05T23:50:36+02:00
+date: 2022-10-12T13:50:36+02:00
 weight: 7
 draft: false
 ---
 
-# Contest machines
+# Image
+The image is based on Ubuntu 22.04 LTS, the following programs/tools are installed.
 
-The contest machine details will be announced later.
+# Contest control system
```

Comment:
Dit kopje voelt hier een beetje vreemd, not?

Reply:
Alsin, je wil die hele kop weghalen?

Reply:
Ahh, will do

Reply:
Done